### DISCUSSION TITLE
Error with AWS S3 using private bucket

### DISCUSSION THREAD

**Discussion by davinyoeun** [oneduo/nova-file-manager#412] [Q&A]
Access to XMLHttpRequest at '
[https://localhost....)
from origin '
http://localhost:8000'
has been blocked by CORS policy: Request header field access-control-allow-methods is not allowed by Access-Control-Allow-Headers in preflight response.

**Comment by Rezrazi** [comment]
I suspect an underlying misconfiguration in the bucket's policies.

Please check S3 cors policies to allow your website to access content.

Happy to help if error is linked to the package.
